corehq/apps/domain/templates/domain/select.html

```diff
@@ -26,17 +20,19 @@ <h2>{% trans "Project Invitations" %}</h2>
           {% endfor %}
         </ul>
       {% endif %}
-    </div>
-    <div class="col-sm-4">
-    </div>
-    <div class="col-sm-4">
-      {% if not restrict_domain_creation or request.user.is_superuser %}
-        {% if not hide_create_new_project %}
-          <aside class="well">
-            <h3>{% trans "Have a new idea?" %}</h3>
-            <a class="btn btn-primary" href="{% url "registration_domain" %}">{% trans "Create a Blank Project" %}</a>
```

Comment:
does this mean you are no longer able to register a domain from that page?

Reply:
yes, you can now only register a domain from the domain dropdown